_data/chains/eip155-5734951.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Jovay Mainnet",
+  "chain": "ETH",
+  "status": "incubating",
+  "rpc": [
+    "https://api.zan.top/public/jovay-mainnet",
+    "https://api.zan.top/node/v1/jovay/mainnet/${ZAN_API_KEY}",
+    "wss://api.zan.top/node/ws/v1/jovay/mainnet/${ZAN_API_KEY}"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "infoURL": "https://jovay.io",
+  "shortName": "jovay",
+  "chainId": 5734951,
+  "networkId": 5734951,
+  "icon": "jovay",
+  "explorers": [
+    {
+      "name": "Jovay Explorer",
+      "url": "https://explorer.jovay.io",
```

Comment:
Looks like this doesn't resolve

Reply:
Sorry, my coworker has fixed it. 🫶 Please try it again.